charts/aws-vpc-cni/templates/clusterrole.yaml

```diff
@@ -33,3 +33,8 @@ rules:
     resources:
       - events
     verbs: ["create", "patch", "list"]
+  - apiGroups:
+      - vpcresources.k8s.aws
+    resources:
+      - cninodes
+    verbs: ["get", "list", "patch"]
```

Comment:
Let's discuss on adding "patch" permission offline.

Reply:
Also we need to remove the `update` permission from node resource right?

Reply:
We need remove node update permission. The problem is we can merge the deletion to master. I have synced with Jeff offline. We may use a feature branch instead.